Core/Core/Models/Utilities.cs

```diff
@@ -32,8 +31,8 @@ public static string HashString(string input, HashingFunctions func = HashingFun
   {
     return func switch
     {
-      HashingFunctions.SHA256 => Crypt.Sha256(input, end: HashLength),
-      HashingFunctions.MD5 => Crypt.Md5(input, end: HashLength),
+      HashingFunctions.SHA256 => Crypt.Sha256(input, length: HashLength),
+      HashingFunctions.MD5 => Crypt.Md5(input, length: HashLength),
```

Comment:
were we using the wrong prop all along? or was `end` just deprecated in favour of `length`

Reply:
These our our functions that I recently exposed the `start` and `end` arguments.
However, I didn't realise that these are not start and end, they are startIndex and length.
Hence my rename here.  

This change hasn't shipped yet, so no damage.